File: Python/String-1/make_tags.py

# given tag and word strings, create the HTML string with tags around the word
def make_tags(tag, word):
    return '<' + tag + '>' + word + '</' + tag + '>'

print(make_tags('i', 'Yay'))
print(make_tags('i', 'Hello'))
print(make_tags('cite', 'Yay'))

File: Java/String-1/makeTags.java

public class string1 {
	
	// given tag and word strings, create the HTML string with tags around the word
	public String makeTags(String tag, String word) {
		
		return "<" + tag + ">" + word + "</" + tag + ">";
		
	}

	public static void main(String[] args) {
		
		string1 o = new string1();
		
		System.out.println(o.makeTags("i", "Yey"));
		System.out.println(o.makeTags("i", "Hello"));
		System.out.println(o.makeTags("cite", "Yay"));
	
	}

}
